pyproject.toml

```diff
@@ -39,15 +39,16 @@ diagnostics = [
     "pandas"
 ]
 beam = ["apache-beam", "gcsfs"]
-lithops = ["lithops[aws] >= 2.7.0"]
+lithops-aws = ["lithops[aws] >= 2.7.0"]
+lithops-gcf = ["lithops[gcf,gcp] >= 2.7.0"]
 modal = [
     "cubed[diagnostics]",
     "modal-client",
     "s3fs",
 ]
 test = [
     "apache-beam",  # beam but not gcsfs as tests use local beam runner
-    "cubed[diagnostics,lithops]",  # modal tests separate due to conflicting package reqs
+    "cubed[diagnostics,lithops-aws]",  # modal tests separate due to conflicting package reqs
```

Comment:
Should we update the tests to also test with GCF?

Reply:
I think unit tests should just test against local Lithops by default, so `"cubed[diagnostics,lithops]"` should be enough here. We can add integration tests for different Lithops cloud services separately.

Reply:
Reverted this change.